onceover.gemspec

```
@@ -17,25 +17,25 @@ Gem::Specification.new do |s| # rubocop:disable Gemspec/RequireMFA
   s.bindir       = 'bin'
   s.executables  = 'onceover'
 
-  s.required_ruby_version = Gem::Requirement.new('>= 2.7')
+  s.required_ruby_version = Gem::Requirement.new('>= 3.2')
 
   # Runtime dependencies, but also probably dependencies of requiring projects
-  s.add_dependency 'backticks', '>= 1.0.2'
-  s.add_dependency 'colored', '>= 1.2'
-  s.add_dependency 'cri', '>= 2.6'
-  s.add_dependency 'deep_merge', '>= 1.0.0'
-  s.add_dependency 'git'
-  s.add_dependency 'logging', '>= 2.0.0'
-  s.add_dependency 'multi_json', '>= 1.10'
-  s.add_dependency 'parallel_tests', ">= 2.0.0"
-  s.add_dependency 'puppet', '>=4.0'
-  s.add_dependency 'puppetlabs_spec_helper', ">= 0.4.0"
-  s.add_dependency 'r10k', '>=2.1.0'
-  s.add_dependency 'rake', '>= 10.0.0'
-  s.add_dependency 'rspec', '>= 3.0.0'
-  s.add_dependency 'rspec_junit_formatter', '>= 0.2.0'
-  s.add_dependency 'rspec-puppet', ">= 2.4.0"
-  s.add_dependency 'terminal-table', '>= 1.8.0'
-  s.add_dependency 'versionomy', '>= 0.5.0'
-  s.add_development_dependency 'voxpupuli-rubocop', '~> 3.0.0'
+  s.add_dependency 'backticks', '~> 1.0'
+  s.add_dependency 'colored', '~> 1.2'
+  s.add_dependency 'cri', '~> 2.6'
+  s.add_dependency 'deep_merge', '~> 1.0'
+  s.add_dependency 'git', '~> 3.1'
+  s.add_dependency 'logging', '~> 2.0'
+  s.add_dependency 'multi_json', '~> 1.10'
+  s.add_dependency 'openvox', '>= 7', '< 9'
+  s.add_dependency 'parallel_tests', '~> 5.3'
+  s.add_dependency 'r10k', '~> 5.0'
+  s.add_dependency 'rake', '~> 13.3'
+  s.add_dependency 'rspec', '~> 3.0'
+  s.add_dependency 'rspec_junit_formatter', '~> 0.6'
+  s.add_dependency 'rspec-puppet', '~> 5.0'
+  s.add_dependency 'terminal-table', '~> 4.0'
+  s.add_dependency 'versionomy', '~> 0.5'
+
+  s.add_development_dependency 'voxpupuli-rubocop', '~> 4.0'
```

Pleaee pin ton4.1.0, minor Versions can pull in newer RoboCop Versions which can cause CI issues

```suggestion
  s.add_development_dependency 'voxpupuli-rubocop', '~> 4.1.0'
```